src/citrine/informatics/predictor_evaluation_result.py

```diff
@@ -0,0 +1,257 @@
+    trial = properties.Integer("trial")
+    fold = properties.Integer("fold")
+    predicted = properties.Mapping(properties.String, properties.Float, "predicted")
+    actual = properties.Mapping(properties.String, properties.Float, "actual")
+    typ = properties.String('type', default='CategoricalPredictedVsActual', deserializable=False)
+
+    def __init__(self, *,
+                 uuid: UUID,
+                 identifiers: Set[str],
+                 trial: int,
+                 fold: int,
+                 predicted: Mapping[str, float],
+                 actual: Mapping[str, float]):
+        self.uuid = uuid
+        self.identifiers = identifiers
+        self.trial = trial
+        self.fold = fold
+        self.predicted = predicted
+        self.actual = actual
+
+
+class PredictedVsActualValue(Serializable["PredictedVsActualValue"], MetricValue):
+    """[ALPHA] List of predicted vs. actual data points.
+
+    Parameters
+    ----------
+    value: List[PredictedVsActual]
+        List of predicted vs. actual data computed during a predictor evaluation.
+        This is a flattened list that contains data for all trials and folds.
+
+    """
+    value = properties.List(properties.Object(PredictedVsActual), "value")
+    typ = properties.String('type', default='PredictedVsActualValue', deserializable=False)
+
+    def __init__(self, value: List[PredictedVsActual]):
+        self.value = value
+
+
+class ResponseMetrics(Serializable["ResponseMetrics"]):
+    """[ALPHA] Set of metrics computed by a Predictor Evaluator for a single response.
+    Results computed for a metric can be accessed by the metric's ``__repr__`` or by the metric itself.
+
+    Parameters
+    ----------
+    metrics: Mapping[str, MetricValue]
+        Metrics computed for a single response, keyed by the metric's ``__repr__``.
+
+    """
+    metrics = properties.Mapping(properties.String, properties.Object(MetricValue), "metrics")
+    typ = properties.String('type', default='ResponseMetrics', deserializable=False)
+
+    def __init__(self, *, metrics: Mapping[str, MetricValue]):
+        self.metrics = metrics
+
+    def __getitem__(self, item):
+        if isinstance(item, str):
+            return self.metrics[item]
+        elif isinstance(item, PredictorEvaluationMetric):
+            return self.metrics[repr(item)]
+        else:
+            raise TypeError("Cannot index ResponseMetrics with a {}".format(type(item)))
+
+
+class PredictorEvaluationResult(PolymorphicSerializable["PredictorEvaluationResult"], metaclass=ABCMeta):
+    """[ALPHA] A Citrine Predictor Evaluation Result represents a set of metrics
+    computed by a Predictor Evaluator.
+
+    """
+
+    @classmethod
+    def get_type(cls, data) -> Type[Serializable]:
+        """Return the subtype."""
+        return {
+            "CrossValidationResult": CrossValidationResult,
+        }[data["type"]]
+
+    @property
+    def evaluator(self) -> PredictorEvaluator:
```

Comment:
How do we feel about converting this and the other abstract methods to properties? (We currently use `@property` to access workflow executions, so a similar pattern is used elsewhere in the SDK.)

Reply:
Sure